File: Python/01 - 10/05 - eulerFive.py
def main():
    exit, num = 0, 1
    while (exit == 0):
        if num % 1 == 0 and num % 11 == 0 and num % 12 == 0 and num % 13 == 0 and num % 14 == 0 and num % 15 == 0 and num % 16 == 0 and num % 17 == 0 and num % 18 == 0 and num % 19 == 0 and num % 20 == 0:
            print(num, "is the smallest number that is divisible by 1 through 20")
            exit = 1
        num = num + 1

if __name__ == "__main__":
    main()

File: C++/01 - 10/05_eulerFive.cpp
// What is the smallest positive number that is evenly divisible by all of the numbers from 1 to 20?
#include <iostream>
int main() {
    bool allDivisible(int number);
    bool evaluating = true;
    int number = 0;
    while(evaluating) {
        number++;
        if(allDivisible(number)) {
            evaluating = false;
        }
    }
    std::cout << number << " is the smallest positive number that is evenly divisible by all of the numbers from 1 to 20.\n";
    return 0;
}
bool allDivisible(int number) {
    for(int i = 1; i <= 20; i++) {
        if(number % i != 0) {
            return false;
        }
    }
    return true;
}